lib/runtimeSettings.js

```diff
@@ -323,6 +323,15 @@ module.exports = {
     runtimeState: {
         enabled: true,
         ui: true
+    },
+    httpAdminMiddleware: function(req,res,next) {
+        res.set("Content-Security-Policy", "frame-ancestors 'self' ${settings.forgeURL}");
+        next()
+    },
+    httpAdminCookieOptions: {
+        sameSite: 'None',
+        secure: true,
```

Comment:
I've fixed the other issue via #240.

Now, back to this one - I can confirm that having *any* of these three values set will break accessing the editor with localfs. So rather than just guarding the `secure` flag, all three should only be set if `forgeURL` starts `https`.

Reply:
That was a big oversight on my end, thank you for spotting it!